waspc/src/Wasp/AI/GenerateNewProject/Plan.hs

```diff
@@ -61,32 +61,35 @@ generatePlan newProjectDetails planRules = do
     appDescriptionBlockText = appDescriptionBlock newProjectDetails
     basicWaspLangInfoPrompt = Prompts.basicWaspLangInfo
     waspFileExamplePrompt = Prompts.waspFileExample
+    prismaFileExamplePrompt = Prompts.prismaFileExample
     rulesText = T.pack . unlines $ "Instructions you must follow while generating plan:" : map (" - " ++) planRules
     planPrompt =
       [trimming|
         ${basicWaspLangInfoPrompt}
 
         ${waspFileExamplePrompt}
 
+        ${prismaFileExamplePrompt}
+
         We are looking for a plan to build a new Wasp app (description at the end of prompt).
 
         ${rulesText}
 
         Plan is represented as JSON with the following schema:
 
         {
-          "entities": [{ "entityName": string, "entityBodyPsl": string }],
+          "models": [{ "modelName": string, "modelBody": string }],
```

Comment:
Here I am tempted to go with `entities`. 
`{ entityName: string, bodyPsl: }` like it was before. Then it is clear these are entities in Wasp and they are implemented via PSL models.

Then we would also update accordingly in the reset of the file (and maybe somewhere else if that pulls that change).

I am not 100% on this, I will leave the decision to you, although I would probably go for it.

Reply:
We are now back to `entities` as a Wasp level concept but we refer to the Prisma models as models, as we discussed offline :D